brooklyn-tosca-transformer/src/test/java/io/cloudsoft/tosca/a4c/Alien4CloudIntegrationTest.java

```diff
@@ -34,8 +35,13 @@
     protected ToscaTypePlanTransformer transformer;
     protected Uploader uploader;
 
+    protected Boolean origFeatureEnablement;
+    
     @BeforeMethod(alwaysRun = true)
     public void setUp() throws Exception {
+    	origFeatureEnablement = BrooklynFeatureEnablement.isEnabled(ToscaTypePlanTransformer.FEATURE_TOSCA_ENABLED);
+    	BrooklynFeatureEnablement.setDefault(ToscaTypePlanTransformer.FEATURE_TOSCA_ENABLED, true);
```

Comment:
I'm not sure we'd ever want an integration test without this enabled, so I'm not sure what rolling back achieves?